tsz-compress/src/v2/constants.rs

```diff
@@ -0,0 +1,13 @@
+pub struct Headers;
```

Comment:
The more idiomatic approach is to have a `mod consts`, then the constants are available within the module. and example of that would be core::f32::consts::PI.

Reply:
Updated in commit [750e1f9](https://github.com/qsib-cbie/tsz/pull/44/commits/750e1f991cbb98bc422568ecce44263066a244b5).